tests/php/ORM/DataObjectSchemaTest.php

```diff
@@ -159,68 +159,94 @@ public function testTableForObjectField()
         );
     }
 
-    public function testFieldSpec()
+    public function provideFieldSpec()
```

Comment:
All of the scenarios in this provider except for the _last one_ come from the original test - I've just refactored it to use a provider so it's clearer what's going on.